### ISSUE TITLE
Minor UI bugs

### ISSUE THREAD

**Issue by tostercx** [QuickNET-Tech/Universal_GTAO_Booster#2]
Nice work on the logos - wanted to have a look at those but you beat me to it :)

Ran into a few problems while testing (both with auto-injection/manual launch w/ Xenos).

1. The new prompt that asks if you want a console spawns under the game for me.
I'm guessing it's created before the game's window is, that's why it happens? Works fine if I inject later.
I didn't notice it at first and was wondering why nothing worked. So I rebooted the game 3x and missed the prompt 3x (lol...)
2. The console doesn't get closed if I inject before the window is created. Tried debugging it, failed. The console API is pretty bad tho, wouldn't be surprised if it was a bug. Hiding it still works FWIW `ShowWindow(consoleWindow, SW_HIDE);`

**Comment by QuickNET-Tech** [comment]
Sounds like WinAPI being a pita. For the message box I can try a different flag on the message box (probably the right solution), or if I can somehow get a handle to the window I can force it on top (a hack if the proper solution doesn't work. As for the console, it wouldn't be the first time I've seen it not work as it should, which is why in the code I'm actually extra forceful and actively destroy the window after freeing the console.

I'll see if I can reproduce asap, thanks.

**Comment by QuickNET-Tech** [comment]
If you're able to pull and build, I've made a few tweaks that should hopefully help. Unfortunately I've not been able to reproduce so...

**Comment by tostercx** [comment]
Nope, both still reliably happen every time for me! (Except now the R* logo patch is applied too late sometimes). Really surprised `DestroyWindow` doesn't do anything.

> Unfortunately I've not been able to reproduce so...

Yeah, if my guess is right, it's timing based, so it would be a PITA to recreate/debug :/
Could be something else on my system as well (running Steam version/windowed if that matters).

It's not too big of a deal - could always fix later if more people complain. Can either disable the console or minimize it as a workaround.

Gonna try debugging it some more later.

**Comment by QuickNET-Tech** [comment]
Tbh this pretty much sums up my experience with the WinAPI, it does what it wants when it wants. 

Reverted 4e4b5b4

**Comment by Oskars999** [comment]
Sveiks, tostercx! Atvaino, ka rakstu offtopic, bet kā iespējams ar tevi sazināties privāti? 
Hi, tostercx! Sorry for the article offtopic, but how I can to contact you privately?
